controllers/usersignup/usersignup_controller_test.go

```diff
@@ -1839,6 +1839,78 @@ func TestUserSignupDeactivatedAfterMURCreated(t *testing.T) {
 		require.Equal(t, "https://registration.crt-placeholder.com", notification.Spec.Context["RegistrationURL"])
 		assert.Equal(t, "userdeactivated", notification.Spec.Template)
 	})
+
+	t.Run("second deactivated notification not created when it already exists", func(t *testing.T) {
```

Comment:
What about a similar test for the second deactivating notification case?

Reply:
Added